src/browsergym/workarena/tasks/base.py

```diff
@@ -64,6 +64,17 @@ def __init__(
         self.timeout = 10000  # ms
 
         self.instance = instance if instance is not None else SNowInstance()
+
+        # Check if workarena-install is done correctly
+        property_name = "workarena.installation.date"
+        try:
+            _ = get_instance_sys_property(self.instance, property_name)
+        except Exception:
+            raise RuntimeError(
+                f"ServiceNow instance is most likey not installed. "
```

Comment:
Suggestion: "WorkArena installation has not been detected in your instance. Please install WorkArena by running `workarena-install`"

Reply:
Perfect!